optuna/serialization.py

```diff
@@ -0,0 +1,20 @@
+import pickle
+
+import optuna
+
+
+def save_study(study: optuna.study.Study, path: str) -> None:
+    study_in_memory = optuna.create_study()
```

Comment:
I'm not confirmed, but maybe.
```suggestion
    study_in_memory = optuna.create_study(directions=study.directions)
```